src/sage/combinat/colored_permutations.py

```diff
@@ -989,6 +1001,18 @@ def __classcall_private__(cls, pi):
             sage: SignedPermutation([2, 1, -3])
             [2, 1, -3]
 
+            sage: SignedPermutation((2,1,-3))
+            [2, 1, -3]
+
+            sage: SignedPermutation(range(1,4))
+            [1, 2, 3]
+
+        TESTS::
+
+            sage: T = SignedPermutation(range(1,4)); type(T)
+            <class 'sage.combinat.colored_permutations.SignedPermutations_with_category.element_class'>
+            sage: SignedPermutation(T)
+            [1, 2, 3]
```

Comment:
```suggestion

            sage: T = SignedPermutation(range(1,4))
            sage: SignedPermutations(3)(T)
            [1, 2, 3]
```
However, this should be in the `_element_constructor_` doctests.